testsuite/sclang/TestLexerBrutal.sc

```diff
@@ -19,68 +16,37 @@ TestLexerBrutal : UnitTest {
 	// (fullLimit+1) to (smallLimit)
 	classvar fullAlphabetStringSizeLimit = 2;
 	classvar smallAlphabetStringSizeLimit = 3;
-	classvar miniAlphabetStringSizeLimit = 5;
+	classvar miniAlphabetStringSizeLimit = 4;
 
 	classvar directory = "brutal_lexer_results/";
 
-	*new {
-		^super.new.initAlphabets;
-	}
+	// set to TRUE if you need `_expected` files
+	makingValidationFiles { ^true; }
 
 	initAlphabets {
 		// init alphabets
 		var alphabets = [
-			fullAlphabet = (-128..-1) ++ (1..127),
-			smallAlphabet = (1..126), // 127 is `DEL`
+			(-128..-1) ++ (1..127),
+			(1..126), // 127 is `DEL`
 			// all punctuation, plus [019AZaz]
-			miniAlphabet = (32..49) ++ (57..65) ++ (90..97) ++ (122..126)
+			(32..49) ++ (57..65) ++ (90..97) ++ (122..126)
 		];
 
 		if(ignoreCaret) {
-			alphabets.do {
+			alphabets = alphabets.collect {
 				arg alphabet;
 				alphabet = alphabet.reject(_==caretAscii);
 			}
 		};
 
-		alphabets.do {
+		alphabets = alphabets.collect {
 			arg alphabet;
 			alphabet = alphabet.collect(_.asAscii).collect(_.asString);
```

Comment:
my pleasure! Just wanted to be sure :) Thanks for these thorough test cases, looking forward to more flexibility in the future!